src/applications/registry.json

```diff
@@ -797,7 +797,7 @@
     "entryName": "your-debt",
     "rootUrl": "/manage-va-debt/your-debt",
     "template": {
-      "vagovprod": false,
+      "vagovprod": true,
```

Comment:
Is this in production now?

Reply:
@Mottie no we're just seeing if this makes the test pass since test run against a prod build